app/design/adminhtml/default/default/template/api/rolesedit.phtml

```diff
@@ -29,8 +29,8 @@
         <span class="field-row">
             <label for="all"><?php echo $this->__('Resource Access') ?></label>
             <select id="all" name="all" onchange="$('resources_container').toggle()" class="select">
-                <option value="0" <?php echo ($this->getEverythingAllowed()?'':'selected="selected"'); ?>><?php echo $this->__('Custom') ?></option>
-                <option value="1" <?php echo ($this->getEverythingAllowed()?'selected="selected"':''); ?>><?php echo $this->__('All') ?></option>
+                <option value="0" <?php echo($this->getEverythingAllowed() ? '' : 'selected="selected"'); ?>><?php echo $this->__('Custom') ?></option>
```

Comment:
Missing space before echo. Is this intentional?

Reply:
No, its not,

Added spaces. Run php-cs-fixer again. Nothing changed. Dont know why space got removed.

Reply:
Last commit added spaces that no changed before .... so it already was inconsistent.

Reply:
Seems you've found a bug.

running php-cs-fixer ...

- w/o custom path fixes (remove) that spaces
- w/ custom path `app/design/` finds nothing - wrong, BUT ...
- w/ custom path `app/` finds issues in code/core/Zend - correct, because directory is ignored in config